nixos/modules/security/acme.nix

```diff
@@ -139,6 +139,18 @@ in
         '';
       };
 
+      production = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          If set to true, use Let's Encrypt's production environment
+          instead of the staging environment. The main benefit of the
+          staging environment is to get much higher rate limits.
+
+          See
+          <literal>https://letsencrypt.org/docs/staging-environment</literal>
+          for more detail.  ''; };
```

Comment:
This is some weird closing of quotes and braces, please keep to the style used in the rest of this file :)